x/evm/types/types.go

```diff
@@ -560,6 +560,47 @@ func CreateApproveContractCallWithMintCommand(
 	}, nil
 }
 
+// DecodeApproveContractCallWithMintParams unpacks the parameters of a approve contract call with mint command
+func DecodeApproveContractCallWithMintParams(bz []byte) (string, string, common.Address, common.Hash, string, *big.Int, error) {
```

Comment:
```suggestion
func DecodeApproveContractCallWithMintParams(bz []byte) (string, string, common.Address, common.Hash, string, sdk.Int, error) {
```
Perhaps wrap the amount in `sdk.Int` for consistency?

Reply:
this PR was created before that. Will update

Reply:
For consistency, it should remain as `*big.Int`, for example because of https://github.com/axelarnetwork/axelar-core/blob/b43155c1dfd2c50a80f1d49604c8f9a1ee9aa4da/x/evm/types/types.go#L1163 and https://github.com/axelarnetwork/axelar-core/blob/b43155c1dfd2c50a80f1d49604c8f9a1ee9aa4da/x/evm/types/types.go#L689